src/NuGet.Core/NuGet.Protocol/Plugins/PluginDiscoverer.cs

```diff
@@ -187,6 +187,11 @@ internal List<PluginFile> GetPluginsInNuGetPluginPaths()
 
             foreach (var path in paths)
             {
+                if (string.IsNullOrEmpty(path))
```

Comment:
Can we just make the split ignore empty?

There's an option for removing empty entries.

Reply:
I see @jeffkl had similar feedback.

Most of the time we just do ignore entry to avoid dealing with the empty or whitespace values